internal/controllers/cluster/cluster_controller.go

```diff
@@ -112,6 +112,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Re
 		}
 
 		// Error reading the object - requeue the request.
+		log.Error(err, "Failed to fetch Cluster")
```

Comment:
```suggestion
```
if we want to add context to the error, let's wrap it in the next line (it will already generate a log line)